src/less/notice/ds6/notice.less

```diff
@@ -13,6 +13,8 @@
 @page-notice-confirmation-background-color: #effef0;
 @page-notice-attention-border-color: @color-r4;
 @page-notice-attention-background-color: #fff5f8;
+@page-notice-light-background-color: #f7f7f7;
```

Comment:
Are we 100% sure this doesn't fall under `@color-grey1: #f5f5f5;`, a color we already have?

Reply:
From what I spoke to design is they changed the background colors slightly in order to match color contrast ratios.
They said all the colors for the guidance/notice modules are different.

Reply:
I mean, it's so stinking close. 🤷‍♂